packages/@react-aria/menu/src/useMenuItem.ts

```diff
@@ -191,6 +192,12 @@ export function useMenuItem<T>(props: AriaMenuItemProps, state: TreeState<T>, re
   }
 
   let onPressStart = (e: PressEvent) => {
+    // TODO: ideally this would be done in useselectableItem but we don't apply that hook's item's props if the node is a menu trigger
+    if (data.shouldUseVirtualFocus && (e.pointerType === 'virtual' || e.pointerType === 'keyboard')) {
+      selectionManager.setFocused(true);
+      selectionManager.setFocusedKey(key);
+    }
+
```

Comment:
would be nice to have this in useSelectableItem, but was mainly added to fix opening the autocomplete submenu on mobile via tap